src/At.vue

```diff
@@ -324,12 +337,13 @@ export default {
           this.closePanel()
         } else {
           const { members, filterMatch, itemName } = this
-          if (!keep && chunk) { // fixme: should be consistent with AtTextarea.vue
+          if (!keep && chunk) {
+            // fixme: should be consistent with AtTextarea.vue
             this.$emit('at', chunk)
           }
-          const matched = members.filter(v => {
+          const matched = members.filter((v) => {
             const name = itemName(v)
-            return filterMatch(name, chunk, at)
+            return filterMatch(name, chunk, at, v)
```

Comment:
Great. It's really helpful. I've taken a quick look at the PR code, and I think the key point lies in this line of change. I'll merge it ASAP, thanks.